roles/bettermarks_proxy/templates/school.conf.j2

```diff
@@ -8,6 +8,7 @@ server{
     proxy_hide_header 'Access-Control-Allow-Credentials';
     add_header 'Access-Control-Allow-Credentials' true;
     # Proxy to the origin
+    proxy_set_header {{ proxy_identification_header }} true;
```

Comment:
I think this header can be set in the server block, so you don't need to declare it so many times for every location block.
see: https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_set_header